packages/mermaid/src/diagrams/class/classDb.ts

```diff
@@ -116,8 +116,8 @@ export const clear = function () {
   commonClear();
 };
 
-export const getClass = function (id: string): ClassNode {
-  return classes[id];
+export const getClass = function (className: string): ClassNode {
+  return classes[className];
 };
```

Comment:
Optional: This would be a big change, but if we use this function everywhere instead of direct dictionary access, we'd only have to add the `addClass` function here, and also handle any future checks easily.

Reply:
I'm not sure if this will work, because there are places where we want to create the class, if it does not exist, and places where we don't want to create the class, if it doesn't exist.

Maybe a `getOrCreateClass()` function might help, though?